### PR TITLE
change response

### PR DESCRIPTION
**Pull request by serhiizalyvshyi**
Changed orderedProducts  data from array to object. Added response 201

### CODE REVIEW

File: controllers/orderController.js [hunk 1]
```diff
@@ -17,26 +17,28 @@ const createOrder = (req, res) => {
...
-        if (err) {
-          console.error(err);
-          res.status(500).json({ error: 'Failed to insert order products' });
-        } else {
-          res.json({ success: true, orderedProducts: orderedProducts });
-        }
+      orderedProducts.forEach((product) => {
+        db.run(insertOrderProductsQuery, [product.order_id, product.product_id, product.quantity, product.name], function (err) {
+          if (err) {
+            console.error(err);
```
Comment (pavloshushkov) [inline controllers/orderController.js:35]: if something will happen how to add human readable error msg to response?

**Review by pavloshushkov — APPROVED** [review]
kinda ok
overall looks great. The comments I left can be turned into separate tasks

File: controllers/orderController.js (hunk 1 above)
Comment (serhiizalyvshyi) [reply controllers/orderController.js:35]: For example: 
const errorMessage = "An error occurred while inserting order products: " + err.message;
res.status(500).json({ error: errorMessage });
      return;

Comment (pavloshushkov) [reply controllers/orderController.js:35]: yeah, but that information must be in response. Not in a console

Comment (serhiizalyvshyi) [reply controllers/orderController.js:35]: Ok

**Comment by pavloshushkov** [comment]
Screenshot 2023-07-11 at 21 43 41
push the button

**Pull request merged**